pkgs/development/libraries/getdns/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, libtool, unbound, libidn, m4, file
+, openssl, doxygen, autoreconfHook, automake, check, libbsd }:
+
+stdenv.mkDerivation rec {
+  pname = "getdns";
+  name = "${pname}-${version}";
+  version = "1.3.0";
+
+  src = fetchurl {
+    url = "https://getdnsapi.net/releases/${pname}-1-3-0/${pname}-${version}.tar.gz";
+    sha256 = "920fa2e07c72fd0e5854db1820fa777108009fc5cb702f9aa5155ef58b12adb1";
+  };
+
+  nativeBuildInputs = [ libtool m4 libbsd ];
+
+  buildInputs = [ unbound libidn openssl doxygen autoreconfHook
```

Comment:
`autoreconfHook`, `automake` and `file` also belongs in `nativeBuildInputs`.

Reply:
I believe `doxygen` is a build-time only dependency as well.